From akpm@linux-foundation.org Mon Feb  5 16:42:19 2007
From: "Robert P. J. Day" <rpjday@mindspring.com>
Date: Mon, 05 Feb 2007 16:41:02 -0800
Subject: USB: Fix misspelled "USBNET_MII" kernel config option.
To: greg@kroah.com
Cc: akpm@linux-foundation.org, rpjday@mindspring.com
Message-ID: <200702060042.l160g56G005248@shell0.pdx.osdl.net>


From: "Robert P. J. Day" <rpjday@mindspring.com>

Fix the misspelling of "USBNET_MII" to "USB_USBNET_MII".

Signed-off-by: Robert P. J. Day <rpjday@mindspring.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/net/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/net/Kconfig
+++ gregkh-2.6/drivers/usb/net/Kconfig
@@ -99,7 +99,7 @@ config USB_USBNET_MII
 
 config USB_USBNET
 	tristate "Multi-purpose USB Networking Framework"
-	select MII if USBNET_MII != n
+	select MII if USB_USBNET_MII != n
 	---help---
 	  This driver supports several kinds of network links over USB,
 	  with "minidrivers" built around a common network driver core
